plugins/chat/assets/javascripts/discourse/components/chat-message-actions-desktop.hbs

```diff
@@ -70,7 +70,11 @@
       }}
         <DropdownSelectBox
           @class="more-buttons secondary-actions"
-          @options={{hash icon="ellipsis-v" placement="left"}}
+          @options={{hash
+            icon="ellipsis-v"
+            placement="left"
+            customStyle="true"
```

Comment:
> This vertical ellipsis button in chat should not have the .btn-default class (should technically have .btn-flat I think, but there's not currently a way to add classes to this kind of select-kit header button?)

I think this can be accomplished by also adding `btnCustomClasses="btn-flat"` to the the options here plus initializing `btnCustomClasses: null,` to the `dropdown-select-box.js`'s `selectKitOptions`.